src/contracts/core/DelegationManager.sol

```diff
@@ -725,6 +728,19 @@ contract DelegationManager is
         return _operatorDetails[operator].delegationApprover;
     }
 
+    /// @inheritdoc IDelegationManager
+    function depositScalingFactor(address staker, IStrategy strategy) public view returns (uint256) {
+        return _depositScalingFactor[staker][strategy].scalingFactor();
```

Comment:
This should return `SlashingLib.scalingFactor()`. Existing stakers who never redeposit after upgrade should read WAD calling this function.